examples/pytorch/Logging/pytorch log model.ipynb

```diff
@@ -0,0 +1,416 @@
+    "    for epoch in range(10):  # number of epochs\n",
+    "        total_loss = 0\n",
+    "        total_accuracy = 0\n",
+    "\n",
+    "        for inputs, labels in train_loader:\n",
+    "            outputs = model(inputs)\n",
+    "            loss = criterion(outputs, labels)\n",
+    "            loss.backward()\n",
+    "\n",
+    "            total_loss += loss.item()\n",
+    "            total_accuracy += calculate_accuracy(outputs, labels)\n",
+    "\n",
+    "        avg_loss = total_loss / len(train_loader)\n",
+    "        avg_accuracy = total_accuracy / len(train_loader)\n",
+    "\n",
+    "        print(f\"Epoch {epoch + 1}, Loss: {avg_loss}, Accuracy: {avg_accuracy}\")\n",
+    "        mlflow.log_metric(\"loss\", avg_loss, step=epoch)\n",
+    "        mlflow.log_metric(\"accuracy\", avg_accuracy, step=epoch)\n",
+    "\n",
+    "    # Log the PyTorch model with the signature\n",
+    "    mlflow.pytorch.log_model(model, \"model\", signature=signature)\n",
+    "\n",
+    "    # Log parameters\n",
+    "    mlflow.log_param(\"epochs\", 10)\n",
+    "    mlflow.log_param(\"batch_size\", 16)\n",
+    "    mlflow.log_param(\"learning_rate\", 0.001)\n",
+    "\n",
+    "print(\"Model training and logging complete.\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Loading the logged model back"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "caac8221-e7bb-4772-9946-acd4c4f331e6",
+     "showTitle": true,
+     "title": "Load back the logged model"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "print(\"Run id from the run above:\", run_id)\n",
+    "\n",
+    "# Later, or in a different script, you can load the model using the run ID\n",
+    "loaded_model = mlflow.pytorch.load_model(f\"runs:/{run_id}/model\")\n",
+    "\n",
+    "# you can now use the loaded model as you would've used the original pytorch model!\n",
+    "loaded_model.eval()\n",
+    "with torch.no_grad():\n",
+    "    sample_input = X_test_tensor[:1]\n",
+    "    loaded_output = loaded_model(sample_input)\n",
+    "    og_output = model(sample_input)\n",
+    "    print(\"Original model output:\", og_output)\n",
+    "    print(\"Loaded model output:\", loaded_output)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "62783c1c-d532-467d-9589-138ee5ef4681",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "\n",
+    "# What you see in the mlflow UI on Databricks\n",
+    "\n",
+    "<img src=\"./screenshots/experiments.png\" style=\"width: 66%\">\n",
```

Comment:
The image link will be broken on our hosted web. Temporarily we can use https://imgur.com/ to host our images, and put the direct link here.